q2_sample_classifier/utilities.py

```diff
@@ -86,6 +89,14 @@ def _load_data(feature_data, targets_metadata):
     return feature_data, targets
 
 
+def _validate_metadata_is_superset(metadata, table):
+    metadata_ids = set(metadata.index.tolist())
+    table_ids = set(table.index.tolist())
+    if not table_ids.issubset(metadata_ids):
```

Comment:
Primarily a stylistic comment, but how about instead of computing this twice we use an intermediate variable:

```python
metadata_ids = set(metadata.index.tolist())
table_ids = set(table.index.tolist())
missing_ids = table_ids.difference(metadata_ids)
if len(missing_ids) > 0:
    raise ValueError(...)
```